cmd/flags.go

```diff
@@ -89,14 +96,8 @@ func DefineFlagVar() *FlagVar {
 		"Enabling verify modules against their signature")
 	flag.BoolVar(&flagVar.enableWebhooks, "enable-webhooks", false,
 		"Enabling Validation/Conversion Webhooks.")
-	flag.BoolVar(&flagVar.enableKcpWatcher, "enable-kcp-watcher", false,
+	flag.BoolVar(&flagVar.enableWatcher, "enable-kcp-watcher", false,
```

Comment:
this flag controls if we enable watcher controller, this kcp indicate the controller is on the KCP, but not in SKR, I'm not against renaming it, but then the actual flag value is still `enable-kcp-watcher`, can you check if we can also rename it without bring additional changes in management config chat? Also please consider improving the description a bit.

Reply:
you are right the actual flag is not renamed and is in use, so i just reverted it.
since we anyways could think about making this option the default to `true` because it's the actual usecase and the special case is false, we can shift this so another PR since it will be more changes and needs an agreement.
also i don't get why it needs to be so specific in name, because there is no `enableSkrWatcher` usecase 😄